mcstatus/responses.py

```diff
@@ -101,6 +101,25 @@ def build(cls, *args, **kwargs) -> Self:
         """
         raise NotImplementedError("You can't use abstract methods.")
 
+    def as_json(self) -> dict:
+        """Return the dataclass as JSON-serializable :class:`dict`.
+
+        Do note that this method doesn't return :class:`string <str>` but
+        :class:`dict`, so you can do some processing on returned value.
+
+        Difference from
+        :attr:`~mcstatus.responses.JavaStatusResponse.raw` is in that,
+        :attr:`~mcstatus.responses.JavaStatusResponse.raw` returns raw response
+        in the same format as we got it. This method returns the response
+        in better format (for example,
```

Comment:
```suggestion
        in a more user-friendly format (for example,
```

Reply:
in a serialiazable format?

Reply:
Done in https://github.com/py-mine/mcstatus/compare/2b9731b13aa5b90f99863488603947026f711fb7..95598e087ac547db2d8c9fa2f4a0c7c050d789d6, thanks!